helm_chart/values.yaml

```diff
@@ -32,6 +32,13 @@ operator:
   - mongodbcommunity
   - mongodbsearch
 
+  # OpenTelemetry configuration
+  opentelemetry:
```

Comment:
Is this supposed to be ever configured by the users? I wonder if we actually need that exposed. I'm a bit worried that it will be confusing why we have `operator.telemetry` and `operator.opentelemetry` in values.yaml. How this is related to each other, is there a possibility for the users to confuse the two and expect no telemetry at all when opentelemetry=false? 

I think we're lacking docs or explanations here. If we don't have good docs/story for it, I'd prefer to not expose the settings here at all.

Reply:
I'm thinking that what we use otel here is just **tracing**. What if we don't mention any "telemetry" but always refer to it as tracing to not confuse with our telemetry gathering?

Reply:
i don't necessarily want to expose this; but how can we use it while not exposing it in our current e2e test infrastructure?

Reply:
can we remove it from the helm's values.yaml and treat it as optional in the template?